pluma/pluma-document-input-stream.c

```diff
@@ -152,6 +162,16 @@ pluma_document_input_stream_class_init (PlumaDocumentInputStreamClass *klass)
 							    G_PARAM_STATIC_NAME |
 							    G_PARAM_STATIC_BLURB |
 							    G_PARAM_CONSTRUCT_ONLY));
+	
+	g_object_class_install_property (gobject_class,
+	                                 PROP_ADD_TRAILING_NEWLINE,
+	                                 g_param_spec_boolean ("add-trailing-newline",
```

Comment:
This seems strange to me. You have a property below to trim the newline, and also a property to add the newline. On top of that there's a gsettings to hide it (but also to keep adding it?). It feels inconsistent.